ext/drbbdup/drbbdup.dox

```diff
@@ -29,3 +29,107 @@
  * OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH
  * DAMAGE.
  */
+
+/**
+***************************************************************************
+***************************************************************************
+\page page_drbbdup Basic Block Duplicator
+
+The \p drbbdup DynamoRIO Basic Block Duplicator Extension provides a mechanism
+to efficiently support different instrumentations of the same basic blocks.
+Although flushing is one viable approach to achieve this functionality,
+it unfortunately also incurs high overheads. Instead, drbbdup duplicates the
+code of the basic block to generate multiple copies and enables a client to
+instrument these copies with different code insertions.
+
+\image html drbbdup_image.svg
+
+The duplication of a basic block is done at
+drmgr's app2app stage. Copies are prepended with a START label,
```

Comment:
Hmm, to me "prepend" and "append" cannot be used this way: they don't take "with", only "to", and thus go the wrong direction here.  I would suggest "prefixed by a START" and "followed by a jump"?

Reply:
Done.